### ISSUE TITLE
Remove VU-2143F and assign VU-58B6

### ISSUE THREAD

**Issue by hfhchan** [hfhchan/irg#184]
Remove VU-2143F and assign VU-58B6

Glyph at VU-2143F matches U+58B6 instead.

![image](https://user-images.githubusercontent.com/8191296/86950057-3528d880-c182-11ea-9c5c-211b1a3c67e4.png)
![image](https://user-images.githubusercontent.com/8191296/86950062-36f29c00-c182-11ea-8093-d4320a523029.png)

**Comment by eisoch** [comment]
You're right. Vietnam team discussed this issue several months ago, they will change it.
















At 2020-07-09 01:19:40, "Henry Chan" <notifications@github.com> wrote:

Remove VU-2143F and assign VU-58B6

Glyph at VU-2143F matches U+58B6 instead.
